spaceship.zsh

```diff
@@ -525,6 +533,30 @@ spaceship_hg() {
     "$SPACESHIP_HG_SUFFIX"
 }
 
+# EMBER
+# Show current version of ember, exception system.
+spaceship_ember() {
```

Comment:
Before merging new feature PR I ask contributors why is their feature useful. Can you prove why is showing Ember version so useful?